xbmc/pvr/PVRManager.h

```diff
@@ -276,8 +276,9 @@ namespace PVR
 
     /*!
      * @brief Let the background thread update the recordings list.
+     * @param clientId The id of the PVR client to update.
      */
-    void TriggerRecordingsUpdate();
+    void TriggerRecordingsUpdate(int clientId = PVR_INVALID_CLIENT_ID);
```

Comment:
Might be more clear to have two functions and skip the default value.

Reply:
Fair.